src/test/java/core/basesyntax/HelloWorldTest1.java

```diff
@@ -0,0 +1,183 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.model.User;
+import core.basesyntax.service.CustomException;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class HelloWorldTest1 {
+    private static RegistrationService registrationService;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @Test
+    void userWithNullLogin_NotOk() {
+        User user = new User();
+        user.setLogin(null);
+        user.setPassword("12345678");
+        user.setAge(22);
+        assertThrows(CustomException.class, () -> {
+            registrationService.register(user);
+        });
+    }
+
+    @Test
+    void nullPassword_NotOk() {
+        User user = new User();
+        user.setLogin("user12345");
+        user.setPassword(null);
+        user.setAge(41);
+        assertThrows(CustomException.class, () -> {
+            registrationService.register(user);
+        });
+    }
+
+    @Test
+    void nullAge_NotOk() {
+        User user = new User();
+        user.setAge(null);
+        Exception exception = assertThrows(CustomException.class, () ->
+                registrationService.register(user));
+        String excepted = "User login,password or age is null";
+        String actual = exception.getMessage();
+        assertTrue(actual.contains(excepted));
+    }
+
+    @Test
+    void threeLettersLogin_NotOk() {
```

Comment:
```suggestion
    void register_invaliLoginLength_NotOk() {
```
better use abstract names.  (check it everywhere)